src/build_targets.rs

```diff
@@ -164,13 +179,19 @@ impl FileNames {
 
                 if target.env == "msvc" {
                     let static_lib = targetdir.join(format!("{lib_name}.lib"));
-                    let impl_lib = targetdir.join(format!("{lib_name}.dll.lib"));
+                    let impl_lib = match use_meson_naming_convention {
+                        false => targetdir.join(format!("{lib_name}.dll.lib")),
+                        true => targetdir.join(format!("{lib_name}.lib")),
+                    };
                     let pdb = Some(targetdir.join(format!("{lib_name}.pdb")));
 
                     (shared_lib, static_lib, Some(impl_lib), pdb, Some(def))
                 } else {
                     let static_lib = targetdir.join(format!("lib{lib_name}.a"));
-                    let impl_lib = targetdir.join(format!("{lib_name}.dll.a"));
+                    let impl_lib = match use_meson_naming_convention {
+                        false => targetdir.join(format!("{lib_name}.dll.a")),
+                        true => targetdir.join(format!("lib{lib_name}.dll.a")),
```

Comment:
Since it's a boolean, it'd be clearer to write something like this:

```suggestion
                    let impl_lib = if use_meson_naming_convention {
                        targetdir.join(format!("lib{lib_name}.dll.a"))
                     } else {
                        targetdir.join(format!("{lib_name}.dll.a"))
```

Reply:
Thanks! Incorporated with commit 5a92c3f567a3b9eebb547d8fedcc7de09b042821.